infra/src/main/kotlin/fi/fta/geoviite/infra/geocoding/Geocoding.kt

```diff
@@ -28,6 +62,12 @@ data class AlignmentAddresses(
     val allPoints: List<AddressPoint> by lazy {
         emptyList<AddressPoint>() + startPoint + midPoints + endPoint
     }
+
+    @get:JsonIgnore
+    val exactMeterPoints: List<AddressPoint> by lazy {
+        // midPoints are even anyhow, so just transform start/end
```

Comment:
Tämä voisi olla myös allPoints.map(::asExactMeter) mutta lähinnä optimointina näin